### PR TITLE
Add Italian Support

### PR DESCRIPTION
**Pull request by sabinrimni**
***References #526***

**Changes proposed:**
* [x] Add

Added Italian Language Support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -289,7 +289,7 @@ The above syntax will create examples for all supported files and folders.
 ## Contributing with translations
 We're looking for people willing to help us translate the extension's messages into [all the languages that `vscode` supports](https://code.visualstudio.com/docs/customization/locales).
 
-We're currently supporting `English`, `German`, `Simplified Chinese` and `Spanish`. If you're willing to help with the translations of the missing languages take a look at [#526](https://github.com/vscode-icons/vscode-icons/issues/526) and get your hands dirty. We'll really appreciate it! ;P
+We're currently supporting `English`, `German`, `Simplified Chinese`, `Italian` and `Spanish`. If you're willing to help with the translations of the missing languages take a look at [#526](https://github.com/vscode-icons/vscode-icons/issues/526) and get your hands dirty. We'll really appreciate it! ;P
```
Comment (JimiC) [inline README.md]: Do alphabetical ordering in the supported languages.

**Comment by JimiC** [comment]
@sabinrimni Perform a lint check (`npm run lint`) and fix the linting issues of the PR.

**Comment by sabinrimni** [comment]
I actually get some errors from `npm run lint`:

ERROR: C:/Users/Vigidis/Documents/GitHub/vscode-icons/src/settings/settingsManager.ts[51, 7]: Calls to 'console.error' are not allowed.
ERROR: C:/Users/Vigidis/Documents/GitHub/vscode-icons/src/settings/settingsManager.ts[41, 26]: Type assertion applied to object literal.
ERROR: C:/Users/Vigidis/Documents/GitHub/vscode-icons/src/icon-manifest/iconGenerator.ts[319, 7]: Calls to 'console.error' are not allowed.
ERROR: C:/Users/Vigidis/Documents/GitHub/vscode-icons/src/icon-manifest/iconGenerator.ts[337, 7]: Calls to 'console.error' are not allowed.
ERROR: C:/Users/Vigidis/Documents/GitHub/vscode-icons/src/example/exampleGenerator.ts[56, 7]: Calls to 'console.error' are not allowed.
ERROR: C:/Users/Vigidis/Documents/GitHub/vscode-icons/src/example/exampleGenerator.ts[65, 7]: Calls to 'console.error' are not allowed.
ERROR: C:/Users/Vigidis/Documents/GitHub/vscode-icons/src/example/exampleGenerator.ts[80, 7]: Calls to 'console.error' are not allowed.
ERROR: C:/Users/Vigidis/Documents/GitHub/vscode-icons/src/example/exampleGenerator.ts[89, 7]: Calls to 'console.error' are not allowed.
ERROR: C:/Users/Vigidis/Documents/GitHub/vscode-icons/src/example/exampleGenerator.ts[139, 5]: Calls to 'console.error' are not allowed.
ERROR: C:/Users/Vigidis/Documents/GitHub/vscode-icons/src/example/exampleGenerator.ts[145, 7]: Calls to 'console.error' are not allowed.
ERROR: test/settings.test.ts[105, 23]: Type assertion applied to object literal.
ERROR: test/iconGenerator/functionality.test.ts[204, 66]: Type assertion applied to object literal.
ERROR: test/iconGenerator/functionality.test.ts[212, 49]: Type assertion applied to object literal.
ERROR: test/iconGenerator/functionality.test.ts[221, 49]: Type assertion applied to object literal.
ERROR: test/iconGenerator/functionality.test.ts[234, 51]: Type assertion applied to object literal.
ERROR: test/iconGenerator/functionality.test.ts[246, 53]: Type assertion applied to object literal.
ERROR: test/iconGenerator/functionality.test.ts[260, 53]: Type assertion applied to object literal.
ERROR: test/iconGenerator/functionality.test.ts[271, 53]: Type assertion applied to object literal.
ERROR: test/iconGenerator/functionality.test.ts[288, 51]: Type assertion applied to object literal.
ERROR: test/iconGenerator/functionality.test.ts[300, 51]: Type assertion applied to object literal.
ERROR: test/iconGenerator/functionality.test.ts[310, 51]: Type assertion applied to object literal.

**Comment by JimiC** [comment]
Install `Yarn` and use that instead of `npm install`.

**Pull request merged**